src/gui/dashboard/overview_screen.py

```diff
@@ -0,0 +1,149 @@
+        self.lbl_size = QLabel()
+        self.btn_update = QPushButton("Save Changes")
+        self.btn_update.setEnabled(False)
+
+    def init_layout(self):
+        layout_main = QVBoxLayout()
+
+        h1 = QHBoxLayout()
+        h1.addWidget(QLabel("Size: "))
+        h1.addWidget(self.lbl_size)
+
+        h2 = QHBoxLayout()
+        h2.addWidget(QLabel("Uploaded: "))
+        h2.addWidget(self.lbl_upload_date)
+
+        h3 = QHBoxLayout()
+        h3.addWidget(QLabel("Modified: "))
+        h3.addWidget(self.lbl_modify_date)
+
+        layout_main.addWidget(QLabel("Name:"))
+        layout_main.addWidget(self.txt_name)
+        layout_main.addWidget(QLabel("Description:"))
+        layout_main.addWidget(self.txt_desc)
+        layout_main.addWidget(QLabel("Tags:"))
+        layout_main.addWidget(self.lst_tags)
+        layout_main.addLayout(h1)
+        layout_main.addLayout(h2)
+        layout_main.addLayout(h3)
+
+        layout_main.addItem(QSpacerItem(1, 1, QSizePolicy.Policy.Minimum, QSizePolicy.Policy.Expanding))
+        layout_main.addWidget(self.btn_update)
+
+        self.setLayout(layout_main)
+
+    def on_selected_file(self, file: FileData):
+        self.txt_name.setText(file.name)
+        self.txt_desc.setText(file.desc)
+        
+        self.lbl_size.setText(f"{file.size} B")
+        self.lbl_upload_date.setText(file.upload_date.strftime('%a %d %b %Y, %I:%M%p'))
+        self.lbl_modify_date.setText(file.last_modified.strftime('%a %d %b %Y, %I:%M%p'))
+
+        self.lst_tags.clear()
+        for tag in file.tags:
+            self.lst_tags.addItem(QListWidgetItem(tag))
+
+
+class OverviewScreen(QWidget):
+    def __init__(self, parent: QTabWidget):
+        QWidget.__init__(self)
+        self.owner = parent
+
+        self.table: QTableWidget
+        self.edit_region: FileEdit
+        self.files: List[FileData] = []
+        self.columns = ['Icon', 'Name', 'Type', 'Date Modified']
+
+        self.init_gui()
+        self.make_layout()
+
+        self.refresh()
+
+    def init_gui(self):
+        self.table = QTableWidget()
+        self.table.setColumnCount(len(self.columns))
+        self.table.setHorizontalHeaderLabels(self.columns)
+        self.table.setSelectionBehavior(QTableView.SelectionBehavior.SelectRows)
+        self.table.setEditTriggers(QAbstractItemView.EditTrigger.NoEditTriggers)
+        self.table.setShowGrid(False)
+        self.table.clicked.connect(self.on_click_item)
+
+        self.edit_region = FileEdit(self)
+        self.edit_region.setVisible(False)
+
+    def make_layout(self):
+        layout_main = QHBoxLayout()
+        
+        hsplitter = QSplitter(Qt.Orientation.Horizontal)
+        hsplitter.addWidget(self.table)
+        hsplitter.addWidget(self.edit_region)
+
+        layout_main.addWidget(hsplitter)
+
+        self.setLayout(layout_main)
+
+    def refresh(self):
+        self.files.clear()
+
+    def refresh(self):
+        self.load_files_from_cloud()
```

Comment:
These functions have the same signature?

Reply:
Removed in [3964463](https://github.com/magley/klau-drive/pull/7/commits/3964463621b0fbc04c939da9bb4fb0b758d3df87).